yb-voyager/test/utils/cmdutils.go

```diff
@@ -0,0 +1,128 @@
+package testutils
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+	testcontainers "github.com/yugabyte/yb-voyager/yb-voyager/test/containers"
+)
+
+func CreateTempExportDir() string {
+	// Create a temporary directory for export inside /tmp
+	exportDir, err := os.MkdirTemp("", "yb-voyager-export")
+	if err != nil {
+		utils.ErrExit("failed to create temp export dir for testing: %v", err)
+	}
+
+	return exportDir
+}
+
+func RemoveTempExportDir(exportDir string) {
+	// Remove the temporary directory
+	err := os.RemoveAll(exportDir)
+	if err != nil {
+		utils.ErrExit("failed to remove temp export dir: %v", err)
+	}
+}
+
+func RunVoyagerCommmand(container testcontainers.TestContainer,
```

Comment:
Interesting! 
I would have thought you'd just call the `assessMigration` function directly. 
In most cases that would be more simpler, however having this ability is also useful as we can test the end-to-end flow.

Reply:
> I would have thought you'd just call the assessMigration function directly.

There was headache of setting all the required variables, and not miss anything.
Running the command seemed a easier approach to me.